views/page-not-found.html

```diff
@@ -0,0 +1,20 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Page not found - File a confirmation statement
+{% endblock %}
+{% block signOut %}{% endblock %}
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        Page not found
+      </h1>
+      <p>If you typed the web address, check it is spelled correctly.</p>
+      <p>If you pasted the web address, check you copied the entire address.</p>
+      <p>If the web address is correct or you selected a link, the page may have moved.
+        <a href="/confirmation-statement">Go to the File a confirmation statement start page</a>.
+
+      </div>
```

Comment:
bottom divs and endblock out of alignment

Reply:
Hi mark, i've pushed up the changes here and for the above but due to the PR already being merged i've had to make a new PR to get the change merged
new PR: https://github.com/companieshouse/confirmation-statement-web/pull/38